src/components/rainbow-toast/ToastDebugSheet.tsx

```diff
@@ -0,0 +1,269 @@
+/* eslint-disable no-plusplus */
+
+import {
```

Comment:
actually I would prefer if we removed `ToastDebugSheet` and the changes related to `isMocked`